CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.4.0 (2022-05-04)
+
+### New features
+
+- Annotations are now inherited from the scheduling task instead of the spawning task for `transfer`. ([#172](https://github.com/pika-org/pika/pull/142))
+- Annotations for bulk regions are now lifted up around the work loop on each worker thread using a scoped annotation. ([#197](https://github.com/pika-org/pika/pull/197))
+- It is now allowed to pass a lambda with `auto` parameters to `then`. ([#182](https://github.com/pika-org/pika/pull/182))
+- A new scheduler based on std::thread is available. ([#200](https://github.com/pika-org/pika/pull/200))
```

Comment:
Maybe:
```suggestion
- A scheduler that spawns `std::thread`s is now available. ([#200](https://github.com/pika-org/pika/pull/200))
```
? I'm trying to think of a way to distinguish this from the existing schedulers which also spawn `std::thread`s, but for a different purpose... I'm not sure this makes it any better though.